atomix/cluster/src/test/java/io/atomix/raft/RandomizedRaftTest.java

```diff
@@ -92,6 +94,62 @@ void raftProperty(
     raftContexts.assertAllLogsEqual();
   }
 
+  @Property(tries = 10, shrinking = ShrinkingMode.OFF, edgeCases = EdgeCasesMode.NONE)
+  void livenessTest(
+      @ForAll("raftOperations") final List<RaftOperation> raftOperations,
+      @ForAll("raftMembers") final List<MemberId> raftMembers,
+      @ForAll("seeds") final long seed)
+      throws Exception {
+
+    setUpRaftNodes(new Random(seed));
+
+    // given - when there are failures such as message loss
+    final var memberIter = raftMembers.iterator();
+    for (final RaftOperation operation : raftOperations) {
+      final MemberId member = memberIter.next();
+      LOG.info("{} on {}", operation, member);
+      operation.run(raftContexts, member);
+    }
+
+    raftContexts.assertOnlyOneLeader();
```

Comment:
:question: I'm wondering about this assert and the assert below plus the comments. 

`// then - eventually a leader should be elected` this was the case before with this assert or not?

Reply:
This assert verifies there are no multiple leaders at the same time. It can be renamed to `assertAtMostOneLeader`.